### PR TITLE
Packagist support

### PR DESCRIPTION
**Pull request by faustbrian**
Adds support for https://packagist.org/ which is used for PHP

### CODE REVIEW

**Comment by faustbrian** [comment]
Added them back, my editor must have removed trailing whitespace on save.

File: libs/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Badgen
 
-Fast badge generating service.  
+Fast badge generating service.
```
[the remark this answers is not in the document]
Comment (faustbrian) [reply libs/index.md]: Added them back, my editor must have removed trailing whitespace on save.

File: libs/live-fns/packagist.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        status: millify(response.package.downloads.daily),
+        color: 'green'
+      }
+    case 'dm':
+      return {
+        subject: 'downloads',
+        status: millify(response.package.downloads.monthly),
+        color: 'green'
+      }
+    case 'favers':
```
Comment (amio) [inline libs/live-fns/packagist.js]: Is this a typo from packagist api? `favers` or `favors`?

Comment (faustbrian) [reply libs/live-fns/packagist.js]: It is what packagist calls it in their API response. `"favers": 11338`

Comment (amio) [reply libs/live-fns/packagist.js]: Hmm, alright, follows it.

**Comment by amio** [comment]
Thanks! @faustbrian
https://badgen.net/#packagist

**Comment by amio** [comment]
Added a `/day`, `/month` suffix to downloads count @faustbrian 
image

**Pull request merged**